src/BPEssentials/Commands/Moderation/UnBan.cs

```diff
@@ -0,0 +1,29 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BPEssentials.Utils;
+using BrokeProtocol.Entities;
+using BrokeProtocol.Utility;
+using BrokeProtocol.Utility.Networking;
+
+namespace BPEssentials.Commands
+{
+    public class UnBan : Command
```

Comment:
Suggest to call the file and class `Unban` instead of `UnBan`.
```suggestion
    public class Unban : Command
```